db/seeds/utils.js

```diff
@@ -1,22 +1,44 @@
+format = require("pg-format");
+const db = require("../connection.js");
+
 exports.convertTimestampToDate = ({ created_at, ...otherProperties }) => {
-	if (!created_at) return { ...otherProperties };
-	return { created_at: new Date(created_at), ...otherProperties };
+  if (!created_at) return { ...otherProperties };
+  return { created_at: new Date(created_at), ...otherProperties };
 };
 
 exports.createRef = (arr, key, value) => {
-	return arr.reduce((ref, element) => {
-		ref[element[key]] = element[value];
-		return ref;
-	}, {});
+  return arr.reduce((ref, element) => {
+    ref[element[key]] = element[value];
+    return ref;
+  }, {});
 };
 
 exports.formatComments = (comments, idLookup) => {
-	return comments.map(({ created_by, belongs_to, ...restOfComment }) => {
-		const article_id = idLookup[belongs_to];
-		return {
-			article_id,
-			author: created_by,
-			...this.convertTimestampToDate(restOfComment),
-		};
-	});
+  return comments.map(({ created_by, belongs_to, ...restOfComment }) => {
+    const article_id = idLookup[belongs_to];
+    return {
+      article_id,
+      author: created_by,
+      ...this.convertTimestampToDate(restOfComment),
+    };
+  });
+};
+
+exports.checkExists = (table, column, value) => {
+  const queryStr = format(
+    `SELECT * FROM %I 
+		WHERE %I = CAST(%L AS INT);`,
```

Comment:
Very nice job putting this into a utils.js, I would argue that you should make a new utils file outside of the seeds folder, though.

Furthermore, now that you are using pg-format for this utils function, it is no longer just a dev dependency - thus, you should change it your package.json

I also don't think you need to CAST here